Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM golang:alpine AS builder
```

Comment:
why are you reverting the docker file back to an ancient version?

Reply:
This Dockerfile was removed on main and I guess I didn't remove it on this branch when merging main into it... Removed now.